docs/reference/commands/node-tool.asciidoc

```diff
@@ -407,6 +429,33 @@ You can also use wildcards to remove multiple settings, for example using
 node$ ./bin/elasticsearch-node remove-settings xpack.monitoring.*
 ----
 
+[float]
+==== Removing custom metadata from the cluster state
+
+If the on-disk cluster state contains custom metadata that prevents the node
+from starting up and loading the cluster state, you can run the following
+commands to remove this custom metadata.
+
+[source,txt]
+----
+node$ ./bin/elasticsearch-node remove-customs snapshot_lifecycle
```

Comment:
Do users have a good way of knowing the name of the custom that should be removed? I.e. `snapshot_lifecycle` here.

Reply:
This API is expert-level, not to be used to remove random stuff. I think the name here will be provided by us for specific support cases where we have previously seen a parse exception. Note that ES still has another way to remove the metadata, namely uninstalling the plugin that defines the corresponding metadata (we just drop the metadata at load time then).